server/forms.py

```diff
@@ -216,7 +216,7 @@ def validate(self):
 
 
 class AssignmentTemplateForm(BaseForm):
-    template_files = FileField('Template Files', [FileRequired()])
+    template_files = FileField('Template Files')
```

Comment:
Removed while testing. Should add it back in. 

We should not be use using templates to enforce required files - will file an issue later.